test/case/ietf_interfaces/verify_all_interface_types/test.py

```diff
@@ -2,8 +2,8 @@
 r"""
 Verify that all interface types can be created
 
-This test verify that all interface types can be created
-and also, tesing setting the configuration in sequal (this
+This test verifies that all interface types can be created
+and also, testing setting the configuration in sequal (this
```

Comment:
Nothing major. Whatever you like more:
"This test verifies that all interface types can be created and also checks the configuration when applied sequentially. This method takes slightly longer than sending the entire configuration at once."

Reply:
I agree this is an improvement, and use it as is. (I also think we can remove the last sentence, but I postpone that for now.)